github/orgs_properties.go

```diff
@@ -48,6 +48,13 @@ type CustomPropertyValue struct {
 	Value        any    `json:"value"`
 }
 
+// CustomPropertyValuesListOptions specifies the optional parameters to the
+// OrganizationsService.ListCustomPropertyValues method.
+type CustomPropertyValuesListOptions struct {
```

Comment:
I propose renaming it to:
```suggestion
type ListCustomPropertyValuesOptions struct {
```

For consistency with https://github.com/google/go-github/blob/ca360914beb3c5f6adb658f8e5c7f081ec97cfcb/github/actions_artifacts.go#L55-L61